webapp/src/js/services/live-list.js

```diff
@@ -45,6 +45,14 @@ angular.module('inboxServices').factory('LiveListConfig',
           if (!c1 || !c2) {
             return;
           }
+
+          const c1Muted = !!c1.muted,
```

Comment:
Unsure if this is made cleaner with the aliasing variables.

Reply:
It's probably not cleaner, but eslint complains otherwise:
> Redundant double negation  no-extra-boolean-cast
Does this matter a lot to you?

Reply:
Doesn't matter.